assets/queries/cloudFormation/aws_bom/ebs/test/positive7.yaml

```diff
@@ -0,0 +1,57 @@
+service: myservice
```

Comment:
I think we should research about the BOM for Serverless resources like api, function, ...